ibis/mapd/tests/test_client.py

```diff
@@ -84,3 +84,35 @@ def test_union_op(alltypes):
     expr = t1.union(t2)
     with pytest.raises(com.UnsupportedOperationError):
         expr.compile()
+
+
+def test_create_table_schema(con):
+    t_name = 'mytable'
+
+    con.drop_table(t_name, force=True)
+
+    schema = ibis.schema([
+        ('a', 'float'),
+        ('b', 'double'),
+        ('c', 'int32'),
+        ('d', 'int64'),
+        ('x', 'point'),
+        ('y', 'linestring'),
+        ('z', 'polygon'),
+        ('w', 'multipolygon')
+    ])
+
+    con.create_table(t_name, schema=schema)
+    t = con.table(t_name)
```

Comment:
This still needs to be inside the `try`, since if it fails then the table won't get deleted.

Reply:
It's only the call to `create_table` that needs to be outside the try.

Reply:
oh I see I got your point! thanks! I did the change!